src/og/quadTree/QuadTreeStrategy.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+import { EPSG3857 } from "../proj/EPSG3857.js";
+
+export class QuadTreeStrategy {
```

Comment:
do we need add test to make sure that the refactor not breaking things?

Reply:
I usually run examples in such cases and look at how it works. Unfortunately, it's not obvious which auto-tests could help in this case, and we have to test manually.